README.md

```diff
@@ -110,10 +110,8 @@ and add to list below:
 If you are new to the group, check out our [New Members Page](NEWMEMBERS.md) and submit a PR to add yourself to the [members list](#stag-members).
 
 ## Related groups
-
-* [Kubernetes Policy Working Group](https://github.com/kubernetes/community/tree/master/wg-policy)
-* [Kubernetes SIG-Auth](https://github.com/kubernetes/community/tree/master/sig-auth)
-* [NIST Big Data WG](https://bigdatawg.nist.gov/)
+- [Known partner groups](./partner-groups/README.md)
+- [Adding or Updating groups](./partner-groups/adding-or-updating-groups.md)  
```

Comment:
I think this goes under the governance folder, and on the `README.md` we probably should give a 1 line to briefly explain partner groups with these links as part of that description

Reply:
Concur with @lumjjb .  This works better under governance.

Reply:
@lumjjb and @TheFoxAtWork are you both saying that the `partner-groups` directory goes under governance directory? AND Remove any reference of partner groups from repo main README and move it to governance/README.md ?

Reply:
Similar to what you have now except everything goes into `governance/partner-groups/` instead of `partner-groups/`